packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -576,14 +575,15 @@ describe('Picker', function () {
 
       act(() => {document.activeElement.blur();});
       act(() => jest.runAllTimers());
+      act(() => jest.runAllTimers());
 
       expect(listbox).not.toBeInTheDocument();
       expect(picker).toHaveAttribute('aria-expanded', 'false');
       expect(picker).not.toHaveAttribute('aria-controls');
       expect(onOpenChange).toBeCalledTimes(2);
       expect(onOpenChange).toHaveBeenCalledWith(false);
 
-      expect(document.activeElement).not.toBe(picker);
+      expect(document.activeElement).toBe(picker);
```

Comment:
I believe this was in error, focus should have been restored because nothing else was on the page